index.html

```diff
@@ -174,7 +174,7 @@ <h3 class="text1">Language</h3>
                 <li>Ruby on Rails</li> 
                 <li>RSPec</li> 
                 <li>CapyBara</li> 
-                <li>Selenium</li>  
+                <li>Selenium</li>   
             </ul>
       </div>
       </div>
```

Comment:
**I'm going to leave the comments here since you do not have commits for your about section**
![image](https://user-images.githubusercontent.com/38088818/185975174-e510eebf-2a34-4c90-9764-8211ed688f0e.png)

- Please make use `display:grid` to layout your `works` section.

![image](https://user-images.githubusercontent.com/38088818/185971192-2c2101f5-accc-4eec-a949-45f7a784e38a.png)

- Please make sure your about section is a `section` tag instead of a `div`.

![image](https://user-images.githubusercontent.com/38088818/185971801-da459d8c-222b-4b4f-bcde-87e553786f2f.png)

- The use of the `hx` must be hierarchal which means that the main heading of the page at the start of the page must be an `h1` element, then subtitles will become `h2`, then `h3`, and so forth.  It would be best if you the `h1` for your main title `Hey There. I'm Diana` then use `h2` or `h3` for your section titles for consistency.